core/src/main/java/com/zextras/carbonio/files/config/FilesConfig.java

```diff
@@ -115,4 +115,27 @@ public String getMailboxUrl() {
   public String getDocsConnectorUrl() {
     return buildUrlFromProperties(DocsConnector.URL, DocsConnector.PORT, "20005");
   }
+
+  public String getMessageBrokerIp() {
```

Comment:
We don't know if it is an IP or an URL:

```suggestion
  public String getMessageBrokerUrl() {
```